src/components/conditional.js

```diff
@@ -45,9 +45,17 @@
           setVisible(initialVisibility);
         }, [checkCondition]);
 
+        useEffect(() => {
+          if (visible) {
+            B.triggerEvent('isTrue', true);
+          } else {
+            B.triggerEvent('isFalse', false);
+          }
+        }, [visible]);
+
         B.defineFunction('Hide', () => setVisible(false));
         B.defineFunction('Show', () => setVisible(true));
-        B.defineFunction('Show/Hide', () => setVisible(s => !s));
+        B.defineFunction('ToggleVisibility', () => setVisible(s => !s));
```

Comment:
Please don't rename the interactions without consulting UX.

Reply:
Ow that means i was maybe a commit behind when i created the branch. Dit not do that on purpose.